src/test/java/racingcar/domain/RacingManagerTest.java

```diff
@@ -0,0 +1,38 @@
+package racingcar.domain;
+
+import static org.assertj.core.api.Assertions.*;
+
+import java.util.List;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RacingManagerTest {
+
+  private Participants participants;
+  private RacingManager racingManager;
+
+  @BeforeEach
+  void init() {
+    participants = new Participants("pobi", "sp", "on");
+    racingManager = new RacingManager(participants, new Round(3), () -> Car.MIN_RUNNABLE_FUEL);
+  }
+
+  @Test
+  void start_우승자_1명() {
+    String expectedLog = "pobi";
+
+    List<Car> cars = participants.cars();
+    cars.get(0).run(Car.MIN_RUNNABLE_FUEL);
+
+    RacingResult racingResult = racingManager.start();
+    assertThat(racingResult.winner()).isEqualTo(expectedLog);
+  }
+
+  @Test
+  void start_우승자_여러명() {
+    String expectedLog = "pobi, sp, on";
+    RacingResult racingResult = racingManager.start();
+    assertThat(racingResult.winner()).isEqualTo(expectedLog);
```

Comment:
```suggestion
    String expectedLog = "pobi, sp, on";
    RacingResult racingResult = racingManager.start();
    
    assertThat(racingResult.winner()).isEqualTo(expectedLog);
```

줄바꿈이 들어가면 테스트 코드 가독성이 더 좋아질 것 같아요~